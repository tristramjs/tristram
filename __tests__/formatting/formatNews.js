// @flow

import newsSiteMapDataMapper from '../../src/formatting/formatNews';

describe('formatNews', () => {
	test('newsSiteMapDataMapper', () => {
		const item = {
			loc: 'br.de/something',
	    news: {
				publication: {
					name: 'Bayerischer Rundfunk',
					language: 'de_de',
				},
				genres: 'Blog',
				publication_date: new Date('1995-12-17T03:24:00'),
				title: 'something',
				keywords: [ 'unrelated', 'bla', 'bla' ],
				stock_tickers: 'nyt',
			},
		};
		expect(newsSiteMapDataMapper(item)).toMatchSnapshot();
	});
});
