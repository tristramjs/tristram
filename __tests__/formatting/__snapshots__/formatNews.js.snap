// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatNews newsSiteMapDataMapper 1`] = `
Object {
  "loc": "br.de/something",
  "news:news": Object {
    "news:genres": "Blog",
    "news:keywords": "unrelated, bla, bla",
    "news:publication": Object {
      "news:language": "de_de",
      "news:name": "Bayerischer Rundfunk",
    },
    "news:publication_date": "1995-12-17T02:24:00.000Z",
    "news:stock_tickers": "nyt",
    "news:title": "something",
  },
}
`;
